### ISSUE TITLE
 how to test your own code

### ISSUE THREAD

**Issue by zhangshabao** [WangYueFt/dcp#5]
Excuse me, I would like to ask you how to test your own code?

**Comment by WangYueFt** [comment]
> Excuse me, I would like to ask you how to test your own code?

Hi,

We're happy to answer any questions. Please ask here.

Thanks,
Yue

**Comment by zhangshabao** [comment]
Using own data for testing, is that data to be converted to a.h5 file?Can I use TXT file directly?

**Comment by WangYueFt** [comment]
You can just change the data loader or directly load your data, then
convert it to torch tensor.

**Comment by zhangshabao** [comment]
Does the requirement of point cloud quantity need to be controlled at 2048?

**Comment by WangYueFt** [comment]
No. If you first train on your data.

**Comment by zhangshabao** [comment]
If I've trained with your data,should the data now be tested in a format consistent with your data?Thank you for your answer！

**Comment by BlankCheng** [comment]
The author has provided download function, it will download modelnet40 automatically. If you want to train with your own data, just change load_data in data.py to fit~

**Comment by zhangshabao** [comment]
Can you share the code?Some questions are not very clear at the beginning.
thank you !

**Comment by zhangshabao** [comment]
> 
> 
> The author has provided download function, it will download modelnet40 automatically. If you want to train with your own data, just change load_data in data.py to fit~

Can you share the code?Some questions are not very clear at the beginning.
thank you !

**Comment by BlankCheng** [comment]
> > The author has provided download function, it will download modelnet40 automatically. If you want to train with your own data, just change load_data in data.py to fit~
> 
> Can you share the code?Some questions are not very clear at the beginning.
> thank you !

ummm I have not trained it and changed the code,  just did some inference on ycb dataset using pretrained model given by author.

**Comment by zhangshabao** [comment]
> 
> 
> > > The author has provided download function, it will download modelnet40 automatically. If you want to train with your own data, just change load_data in data.py to fit~
> > 
> > 
> > Can you share the code?Some questions are not very clear at the beginning.
> > thank you !
> 
> ummm I have not trained it and changed the code, just did some inference on ycb dataset using pretrained model given by author.

I've trained with the author's data set, and I think my data should be converted to match it.Confusion about what that data set contains.

**Comment by BlankCheng** [comment]
> > > > The author has provided download function, it will download modelnet40 automatically. If you want to train with your own data, just change load_data in data.py to fit~
> > > 
> > > 
> > > Can you share the code?Some questions are not very clear at the beginning.
> > > thank you !
> > 
> > 
> > ummm I have not trained it and changed the code, just did some inference on ycb dataset using pretrained model given by author.
> 
> I've trained with the author's data set, and I think my data should be converted to match it.Confusion about what that data set contains.

http://modelnet.cs.princeton.edu/ Modelnet40 is described in this link. "data" in load_data is CAD point cloud , maybe a N * 3 matrix, "label" is its category, a number from 0 to 39. If you want to train the classification with your own data, just fit in this format of "data" and "label", no need to save it in h5.

**Comment by zhangshabao** [comment]
> 
> 
> > > > > The author has provided download function, it will download modelnet40 automatically. If you want to train with your own data, just change load_data in data.py to fit~
> > > > 
> > > > 
> > > > Can you share the code?Some questions are not very clear at the beginning.
> > > > thank you !
> > > 
> > > 
> > > ummm I have not trained it and changed the code, just did some inference on ycb dataset using pretrained model given by author.
> > 
> > 
> > I've trained with the author's data set, and I think my data should be converted to match it.Confusion about what that data set contains.
> 
> http://modelnet.cs.princeton.edu/ Modelnet40 is described in this link. "data" in load_data is CAD point cloud , maybe a N * 3 matrix, "label" is its category, a number from 0 to 39. If you want to train the classification with your own data, just fit in this format of "data" and "label", no need to save it in h5.

Is it convenient to leave your email?Can we continue our conversation?My QQ is 1047253497.Thank you~
